skills/src/lasr_skills/detect_pointing.py

```diff
@@ -0,0 +1,17 @@
+import rospy
+import smach
+from sensor_msgs.msg import Image
+
+from lasr_vision_msgs.srv import PointingDirection
+
+class PointingDetecor(smach.State):
+    def __init__(self):
+        smach.State.__init__(self, outcomes=['succeeded', 'failed'], output_keys=['direction'])
+        self.service = rospy.ServiceProxy('/pointing_detection_service', PointingDirection)
+        self.image_raw = rospy.wait_for_message('/xtion/rgb/image_raw', Image)
+
+    def excute(self, userdata):
+        resp = self.service(self.image_raw)
+        userdata.direction = resp.direction
+
+        return 'succeeded' if resp.direction != 'Err' else 'failed'
```

Comment:
I'm going to add a separate issue to make this more in-line with our other skill implementations

Reply:
#155